alembic/versions/c6250555a36c_.py

```diff
@@ -0,0 +1,409 @@
+"""
+
+Revision ID: c6250555a36c
+Revises: d7c2f99cd14d
+Create Date: 2020-04-06 09:34:50.929724
+
+"""
+import enum
+from datetime import datetime, timezone
+from json import loads
+from os import getenv
+from copr.v3 import CoprNoResultException, Client
+from celery.backends.database import Task
+from redis import Redis
+from typing import TYPE_CHECKING, Union, List
+from persistentdict.dict_in_redis import PersistentDict
+
+from alembic import op
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    DateTime,
+    ForeignKey,
+    Enum,
+    orm,
+)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import Session, relationship
+from sqlalchemy.types import PickleType
+
+from packit_service.service.events import InstallationEvent
+from packit_service.constants import WHITELIST_CONSTANTS
+
+# revision identifiers, used by Alembic.
+revision = "c6250555a36c"
+down_revision = "d7c2f99cd14d"
+branch_labels = None
+depends_on = None
+
+if TYPE_CHECKING:
+    Base = object
+else:
+    Base = declarative_base()
+
+
+# Redis models
+class Model:
```

Comment:
Can we rename that to `RedisModel`?

(I was a bit confused about the following classes what models are they -- postgres x redis.)

Reply:
Maybe also rename the following following classes so the upgrade code can be more readable.